.github/workflows/rhub.yaml

```diff
@@ -95,10 +95,11 @@ jobs:
           job-config: ${{ matrix.config.job-config }}
 
       - name: Prepare DESCRIPTION file
-        uses: insightsengineering/setup-r-dependencies@main
+        uses: insightsengineering/setup-r-dependencies@fix_internal_repo
```

Comment:
```suggestion
        uses: insightsengineering/setup-r-dependencies@main
```

Reply:
Yes, good idea, we can tag the action after https://github.com/insightsengineering/setup-r-dependencies/pull/3 is merged.

Reply:
Just merged this. You can now proceed with tagging and using tags here

Reply:
`v1` and `v1.0.0` of `setup-r-dependencies` is now available [here](https://github.com/insightsengineering/setup-r-dependencies/releases/tag/v1.0.0).